util/walk.js

```diff
@@ -285,7 +285,23 @@
     c(node.object, st, "Expression");
     if (node.computed) c(node.property, st, "Expression");
   };
-  base.Identifier = base.Literal = base.ExportDeclaration = base.ImportDeclaration = ignore;
+  base.ExportDeclaration = function (node, st, c) {
+    c(node.declaration, st);
+  };
+  base.ImportDeclaration = function (node, st, c) {
+    c(node.source, st, 'Literal');
+    node.specifiers.forEach(function (specifier) {
+      c(specifier, st);
+    });
+  };
+  base.ImportSpecifier = function (node, st, c) {
+    if (node.name) c(node.name, st, 'Identifier');
+    else c(node.id, st, 'Identifier');
```

Comment:
In the cases like `import id from 'foo';` it feels only natural that we should walk into `id` just like we do for `var id = 'foo';` but I've chosen not to walk into `id` in cases like `import {id as foo} from foo` as that seems more similar to cases like `foo.id` where we would not walk into `id`.